src/clj/game/core/costs.clj

```diff
@@ -366,7 +366,7 @@
      :choices {:all true
                :max (value cost)
                :card #(and (installed? %)
-                           (not (same-card? % card))
+                           (not (partial same-card? % card))
```

Comment:
I don't know what you think this does, but it's not correct lol. `partial` returns a function, so this returns `false` always. What is the goal?

Reply:
🤦